src/unity/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -32,9 +32,11 @@ namespace one_shot_object_detection {
  */
 class ParameterSampler {
```

Comment:
Really depends on how picky you want me to be. I think it looks fine personally. Wondering is this only going to be used in this file? Is that the reason for not having this class in a header file?

Reply:
Fixed!